src/diffusers/pipelines/wan/pipeline_wan_i2v.py

```diff
@@ -113,9 +113,11 @@ def retrieve_latents(
     latents_mean: torch.Tensor,
     latents_std: torch.Tensor,
     generator: Optional[torch.Generator] = None,
-    sample_mode: str = "sample",
+    sample_mode: str = "none",
 ):
-    if hasattr(encoder_output, "latent_dist") and sample_mode == "sample":
+    if hasattr(encoder_output, "latent_dist") and sample_mode == "none":
+        return (encoder_output.latent_dist.mean - latents_mean) * latents_std
+    elif hasattr(encoder_output, "latent_dist") and sample_mode == "sample":
```

Comment:
```suggestion
    else:
           raise AttributeError("Could not access latents of provided encoder_output")
```
we can remove rest of code here